src/config/i18n/defaults.json

```diff
@@ -371,6 +371,11 @@
     },
     "session_triggered": "Started somebody else in \"[[flow]]\""
   },
+  "spell_checker": {
+    "main_description": "This feature will automatically correct typos and misspellings by users in multiple languages. The language for the spell checker is based on the language set for the user. Corrections occur before the categorization rules. For example, if user types “thankks” it would be corrected to “thanks.” The word “thanks” would then be used for the categorization. The corrected text is available as a flow variable with @flow.variable.corrected",
```

Comment:
That's a minor fix: `@results.variable.corrected` rather than `@flow.variable.corrected`